packages/modules/imodel-browser/src/containers/ITwinGrid/ITwinTile.tsx

```diff
@@ -2,47 +2,47 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
-import "./ProjectTile.scss";
+import "./ITwinTile.scss";
 
 import { Badge, Tile, TileProps } from "@itwin/itwinui-react";
 import React from "react";
 
-import ProjectIcon from "../../images/project.svg";
-import { ProjectFull } from "../../types";
+import ITwinIcon from "../../images/itwin.svg";
```

Comment:
I think the icon should change for iTwins (opposed to only project, I'll see with @FlyersPh9 if we can have something short term available for the major release)

Reply:
sure, let me know when you have new icon

Reply:
Will do